### ISSUE TITLE
RRP SessionError: code 0x2

### ISSUE THREAD

**Issue by CT-H00K** [ly4k/Certipy#24]
Hi, I just tried certipy and I have some issues to find and get bloodhound output. It correctly finds multiple certificate templates and 4 certificate authorities, then tries to connect to all of them, 3 work, and one fails with the following: 

```
[-] Got error: RRP SessionError: code: 0x2 - ERROR_FILE_NOT_FOUND - The system cannot find the file specified.
Traceback (most recent call last):
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\certipy\entry.py", line 62, in main
    actions[options.action](options)
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\certipy\find.py", line 813, in entry
    find.find()
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\certipy\find.py", line 156, in find
    edit_flags, request_disposition, security = self.get_ca_security(
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\certipy\find.py", line 686, in get_ca_security
    policy_key = rrp.hBaseRegOpenKey(
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\impacket-0.9.24-py3.10.egg\impacket\dcerpc\v5\rrp.py", line 874, in hBaseRegOpenKey
    return dce.request(request)
  File "C:\Users\username\.virtualenvs\Certipy-q9BFJWE2\lib\site-packages\impacket-0.9.24-py3.10.egg\impacket\dcerpc\v5\rpcrt.py", line 880, in request
    raise exception
impacket.dcerpc.v5.rrp.DCERPCSessionError: RRP SessionError: code: 0x2 - ERROR_FILE_NOT_FOUND - The system cannot find the file specified.
```
The CA server has to do something with skype for business I guess, don't know if that helps. 

Does it actually have to enumerate all servers?

**Comment by ly4k** [comment]
Hello
I will fix this in the next update. Probably today. This is a bug, since we should be able to continue. It's actually implemented, but the error is not handled

**Comment by ly4k** [comment]
Fixed in version 2.0.1.
